### DISCUSSION TITLE
Replacing Google Groups with GitHub Discussions

### DISCUSSION THREAD

**Discussion by giovtorres** [PySlurm/pyslurm#211] [General]
Now that GitHub has an integrated Discussions feature for projects, does the community feel that this would be a better place to get support instead of the existing PySlurm Google Group?

Would love to get feedback from the community! Thanks.

**Comment by cjw296** [comment]
It seems the last activity on the mailing was also 25th Nov '21, so it's unclear... Is it possible to get email notifications from GitHub discussions?
